12.nadeo-importer/03.how-to-make-a-specularmap-diffusemap-for-objects/docs.md

```diff
@@ -29,11 +49,13 @@ For selfmade textures on objects we need _TDNS_ textures:
 
 ![](http://trackmania-carpark.com/imagespark/up4/52cea5bd6a098.jpg)
 
-* DXT1 RGB 4bpp no alpha
+* DXT1 RGB 4bpp no alpha (care exceptions)
 * 2D texture
 * Generate MIP maps
 
-This is the base texture which is visible, no alpha channel here! Save as `filename_D`.
+This is the base texture which is visible, normally there is no alpha channel. Save as `filename_D`.
+To check the exceptions open the original file in Photoshop or Gimp and check for alpha channel.
```

Comment:
I see you probably meant this with the "care exceptions" above, but you might be able to write it in a different, more concise way. I'd rewrite it a bit, but I don't really understand what you mean with "you need it also for your modified/created file".